src/expr/proof_node_updater.cpp

```diff
@@ -22,6 +22,11 @@ namespace CVC4 {
 ProofNodeUpdaterCallback::ProofNodeUpdaterCallback() {}
 ProofNodeUpdaterCallback::~ProofNodeUpdaterCallback() {}
 
+bool ProofNodeUpdaterCallback::shouldContinue(std::shared_ptr<ProofNode> pn)
```

Comment:
Can we make this a flag on shouldUpdate so that it is parallel to how the shouldContinue flag works on update?